DTS-STN/Scrum-Poker
# Removed the 100 card from card list

### Description

Quick PR to remove the 100 card from the card list

### What to test for/How to test

1. Pull in branch
2. Type `npm run dev`
3. Home page should no longer show the 100 card
